### PR TITLE
Update graphs-in-3-clicks.html

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: lang/en/texts/graphs-in-3-clicks.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 <h1>Advanced Search and Charts</h1>
 
-<a href="https://world.openfoodfacts.org/cgi/search.pl?graph=1" title="Get the app" class="button success">Skip the tutorial</a>
+<a href="https://world.openfoodfacts.org/cgi/search.pl?graph=1" title="Skip the tutorial" class="button success">Skip the tutorial</a>
```
Comment (hangy) [inline lang/en/texts/graphs-in-3-clicks.html]: The `title` seems redundant as the link/button text is the same, but OK.

File: po/common/common.pot [hunk 2]
```diff
@@ -2234,8 +2234,7 @@ msgstr ""
 
 msgctxt "search_graph_blog"
 msgid ""
-"<p>→ learn more about <<site_name>> graphs: <a href=\"https://fr.blog."
-"openfoodfacts.org/news/des-graphiques-en-3-clics\">Graphs in 3 clicks</a> "
+"<p>→ learn more about <<site_name>> graphs: <a href=\"https://world.openfoodfacts.org/graphs-in-3-clicks\">Graphs in 3 clicks</a> "
```
Comment (hangy) [inline po/common/common.pot]: Could this use `/graphs-in-3-clicks` instead? Having the openfoodfacts.org domain in `common` translations sounds not that useful. 🙂

Comment (teolemon) [inline po/common/common.pot]:
```suggestion
"<p>→ learn more about <<site_name>> graphs: <a href=\"/graphs-in-3-clicks\">Graphs in 3 clicks</a> "
```

Comment (teolemon) [reply po/common/common.pot]: @hangy @stephanegigandet it's invoked from : https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=4018077604218

**Comment by teolemon** [comment]
fix commited. good for you @hangy @stephanegigandet ?

**Pull request merged**